docs/Getting-Started-GPU.md

```diff
@@ -1,15 +1,14 @@
-# Getting Started on CUDA
+# Getting Started on GPU
 
-**Note** that right now CUDA support is very preliminary and Levanter does not work on a multi-machine setup. (We think this is just a dependency/environment issue.)
+**Note** that right now GPU support is very preliminary and Levanter does not work on a multi-machine setup. (We think this is just a dependency/environment issue.)
 
 We only test on Ampere (e.g. A100s or 30xx series)  GPUs. If it works with Jax it should work, though.
 
 ## Installation
 ### Setting up Conda Environment
 ```bash
 # 3.11 is too new for tensorstore
-# this doesn't actually install pytorch, but it bundles cuda dependencies nicely
-conda create --name levanter -c pytorch -c nvidia pytorch-cuda=11.7 python~=3.10
```

Comment:
including `-c pytorch and -c nvidia pytorch-cuda=11.7` was causing issues when I was installing jax with cuda and then when I was installing levanter, but without them everything seems to be running correctly.